pkg/collector/upgrade/v0_104_0_test.go

```diff
@@ -56,7 +56,7 @@ func Test0_104_0Upgrade(t *testing.T) {
 		t.Errorf("expect err: nil but got: %v", err)
 	}
 	assert.EqualValues(t,
-		map[string]string{"feature-gates": "-component.UseLocalHostAsDefaultHost"},
+		map[string]string{},
```

Comment:
shouldn't this still be true?

Reply:
maybe not if the test is running _all_ the upgrades...

Reply:
y, I think the Problem is that the upgrade does not stop at 104.